k8s/namespaces/wa/wa-task-batch-initiation-failure/aat-01.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-initiation-failure
+spec:
+  values:
+    job:
+      schedule: "5-59/10 * * * *"
```

Comment:
Are schedules to be different between aat 00 & 01?

Reply:
they are different one of them starts from 0 other one starts from 5
"0-59/10 * * * *"
"5-59/10 * * * *"